src/main/io/osd.c

```diff
@@ -587,6 +604,14 @@ void osdDrawElements(void)
         osdDrawSingleElement(OSD_GPS_LON);
     }
 #endif // GPS
+
+#ifdef USE_ESC_SENSOR
+  if (feature(FEATURE_ESC_SENSOR))
+  {
```

Comment:
For consistency's sake, please use K & R style braces (brace on newline for functions, but on the same line for everything else.